core/templates/pages/skill-editor-page/editor-tab/skill-description-editor/skill-description-editor.directive.ts

```diff
@@ -36,11 +38,12 @@ angular.module('oppia').directive('skillDescriptionEditor', [
         '/pages/skill-editor-page/editor-tab/skill-description-editor/' +
         'skill-description-editor.directive.html'),
       controller: [
-        '$scope', 'EVENT_SKILL_REINITIALIZED',
-        function($scope, EVENT_SKILL_REINITIALIZED) {
+        '$scope',
+        function($scope) {
           var ctrl = this;
-          $scope.MAX_CHARS_IN_SKILL_DESCRIPTION = (
-            MAX_CHARS_IN_SKILL_DESCRIPTION);
+          ctrl.parentSubscription = new Subscription();
+          $scope.MAX_CHARS_IN_SKILL_DESCRIPTION =
```

Comment:
Wrap inside `(..)` in you have statment spanning multiple lines. Break after `(` if required.

Reply:
Done!